scripts/pwned-passwords.sh

```diff
@@ -5,66 +5,30 @@ set -eu
 submit_to_s3='false'
 pwned_directory="pwned_passwords"
 number_of_passwords=3000000
-pwned_url="https://downloads.pwnedpasswords.com/passwords/pwned-passwords-sha1-ordered-by-count-v8.7z"
-pwned_7z="${pwned_directory}/pwned-passwords.7z"
 pwned_file="${pwned_directory}/pwned-passwords.txt"
```

Comment:
Not sure it's too consequential since there's a few steps to updating this in production involving moving files around, but the filename here doesn't match what the application consumes (dash-delimited vs. underscores):

https://github.com/18F/identity-idp/blob/f2d1559b5a37e02aa5a7a81e12248c1194c3c227/config/application.yml.default#L256